orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -326,7 +365,32 @@ func (m *swiftDialogMDMMigrator) waitForUnenrollment() error {
 }
 
 func (m *swiftDialogMDMMigrator) renderMigration() error {
-	message, flags, err := m.getMessageAndFlags()
+	log.Debug().Msg("checking current enrollment status")
+	isCurrentlyManuallyEnrolled, err := profiles.IsManuallyEnrolledInMDM()
+	if err != nil {
+		return err
+	}
+
+	// Check what kind of migration was in progress, if any.
+	previousMigrationType, err := m.mrw.GetMigrationType()
+	if err != nil {
+		log.Error().Err(err).Msg("getting migration type")
+	}
+
+	isManualMigration := isCurrentlyManuallyEnrolled || previousMigrationType == constant.MDMMigrationTypeManual
+	isADEMigration := previousMigrationType == constant.MDMMigrationTypeADE
```

Comment:
⚠️ important: this is a lingering question from before: if there's an error getting the seems like we could have different behaviors depending on `isCurrentlyManuallyEnrolled` and other factors below.

any reason to not return an error? `GetMigrationType` handles `os.ErrNotExist`

Reply:
No reason to not return an error, I think I just forgot.